src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoAbbreviatedMonthGenitiveNames.cs

```diff
@@ -33,5 +33,14 @@ public void AbbreviatedMonths_Set_Invalid()
             // DateTimeFormatInfo.InvariantInfo is read only
             Assert.Throws<InvalidOperationException>(() => DateTimeFormatInfo.InvariantInfo.AbbreviatedMonthGenitiveNames = new string[] { "1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "" });
         }
+        
+        [Fact]
```

Comment:
You need to add `[SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]` here to skip the test on .NET Framework. The bug is not fixed on .NET Framework, and the test needs to be disabled there to make the CI green.

Reply:
@jkotas Thank you